wdn/templates_4.1/less/layouts/share.less

```diff
@@ -183,6 +183,18 @@
     }
 }
 
+/* the share widget dropdown needs some focus love because it's icon is absolutely positioned */
+.wdn-dropdown-widget-label.focused .wdn-icon-share {
+	/* firefox */
+	outline: 1px dotted #212121;
+	/* -webkit-focus-ring-color = '#5B9DD9' */
```

Comment:
what about it? The icon is absolutely positioned, not the label (which usually gets the outline). So, when the control was focused, the label was being outlined but had no width (resulting in a line in some browsers, and no outline in others). This simply tries to simulate the default outline styles in firefox and webkit based browsers. 

This all stems from using the 'pure css' dropdown pattern, which utilizes a checkbox to track the visibility state. Browsers don't give focus to labels, so in order to provide a visual signal that you are current focused on something, we have to do this weird hack.

I'm all for ditching the 'pure css' dropdown pattern we have going on.